### DISCUSSION TITLE
Scss function

### DISCUSSION THREAD

**Discussion by RenaldoJay** [phcode-dev/phoenix#1044] [Q&A]
For some reason unknown to me, the information I have entered in the scss file is not transferred. The file is linked via an html file. I am a beginner, maybe someone can give me some advice.

Kind regards

Jason

**Comment by abose** [comment]
Scss cannot be directly integrated into html files. Scss are usually converted to CSS files and that CSS file is usually linked in the HTML file. Please see https://sass-lang.com/guide

**Reply by RenaldoJay** [reply]
Hi abose, thank you for taking the time to answer me. So then this function is not built into the Phoenix Code Editor? I searched the plugins and found nothing.

**Reply by abose** [reply]
It is not built-in phcode currently, but it may be available as an extension in https://brackets.io .
